saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/web/authentication/logout/OpenSamlLogoutRequestResolver.java

```diff
@@ -102,7 +102,8 @@ public OpenSamlLogoutRequestSpec resolveLogoutRequest(HttpServletRequest request
 			RelyingPartyRegistration registration, Authentication authentication) {
 		return new OpenSamlLogoutRequestSpec(registration)
 				.destination(registration.getAssertingPartyDetails().getSingleLogoutServiceLocation())
-				.issuer(registration.getEntityId()).name(authentication.getName());
+				.issuer(registration.getEntityId())
+				.name(authentication.getName());
 	}
 
 	public class OpenSamlLogoutRequestSpec implements Saml2LogoutRequestSpec<OpenSamlLogoutRequestSpec> {
```

Comment:
Does it make sense to allow users to conveniently change the destination, issuer and name? 
Is there a use case where they would want those values to be different than what they set up in the `RelyingPartyRegistration`?